includes/wikia/api/TvApiController.class.php

```diff
@@ -213,7 +213,7 @@ protected function getConfigById( $wikiId, $articleId ) {
 	protected function findSeries( $seriesName, $lang, $quality = self::DEFAULT_QUALITY ) {
 		//check exact match on series first
 		$result = $this->exactMatchOnSeries( $seriesName, $lang );
-		if ( $result == null ) {
+		if ( $result == null || $result[ 'quality' ] < $quality  ) {
```

Comment:
Let me recap this (a comment would be lovely!):
Current version:
1) We are searching for an exact match on series. 
1a) If we don't find it, we do a regular search.
2) If we have result from exact match, or did fallback to query, we check article quality and return only if threshold is met.

Now, if I understand your change correctly:
1) We are searching for an exact match on series. 
1a) If we don't find it, we do a regular search.
1b) If we find it, but it does not meet the quality threshold, we do a regular search
2) If we have result from exact match, or did fallback to query, we check article quality and return only if threshold is met.

The way I see it is that we now introduce a fallback that can affect the results i.e. in Aida; Imagine we have a low-quality (i.e. 15) static mapping of series description and we have a better one (i.e. 98) on our site. 
Should we introduce this change, this will mean that for queries with AQ threshold of 10 we will return the low-quality result, but for threshold above 15 we will return the high-quality one.

If my reasoning above is correct, I think we should take a different path to ensure we return either the best result all the time, or static mapping (or nothing) in other cases, as we need to remember that a good quality article about wrong series is worse than not getting anything.